presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -504,6 +505,11 @@ public SystemSessionProperties(
                         "Skip redundant sort operations",
                         featuresConfig.isSkipRedundantSort(),
                         false),
+                booleanProperty(
+                        DERIVE_FULL_JOIN_PROPERTIES,
```

Comment:
If we are afraid it might have a bug, then I would call property name `experimental` so that we can easily remove it later.

Reply:
`experimental.` property should not be turned on by default.